resources/community/devhubOnMinishift.md

```diff
@@ -0,0 +1,91 @@
+ #Setup Docker in localhost
+####In case you want to run Devhub on local Minishift:
+
+ Which means you will not Have a nodejs version 10 and s2i-caddy image from Redhat in 'openshift' namespace.
+ 
+ #####You will need:
+ 
+ * Minishift
+ * Docker
+ * Access permission to console.pathfinder.gov.bc/openshift namespace.
+ 
+ 
+####Step 1: Download and config Minishift
+Ensure that you download and install [VirtualBox](https://www.virtualbox.org/wiki/Downloads) before using the embedded drivers.
+Install  before install minishift.
+
+Official instruction can be found [here]( https://docs.okd.io/latest/minishift/getting-started/installing.html).
+
+
+Use the following command to install and config the latest version of the driver with Homebrew on your computer:
+```Code
+brew install docker-machine-driver-xhyve
+sudo chown root:wheel $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+sudo chmod u+s $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
+```
+Install minishift through Homebrew:
+```code
+brew cask install minishift
+```
+The --vm-driver VirtualBox flag will need to be given on the command line each time the minishift start command is run. For example:
+```code
+$ minishift start --vm-driver virtualbox
+```
+ #### Install Docker
+ Docker will be required to pull image from Bcgov openshift and push to your local minishift
+ 
+You can download from [here](https://docs.docker.com/docker-for-mac/install/).
+Make sure that you have the Docker client binary installed on your machine.
+
+###Pull nodejs image
+
+In order to configure your console to reuse the Minishift Docker daemon, follow these steps:
+* Start Minishift with the `minishift start` command.
+* Run the`minishift docker-env` command to display the command you need to type into your shell to configure your Docker client. The command output will differ depending on OS and shell type.
+```command
+$ minishift docker-env
+export DOCKER_TLS_VERIFY="1"
+export DOCKER_HOST="tcp://192.168.99.101:2376"
+export DOCKER_CERT_PATH="/Users/john/.minishift/certs"
+export DOCKER_API_VERSION="1.24"
+# Run this command to configure your shell:
+# eval $(minishift docker-env)
+```
+* Run this command to configure your shell: `eval $(minishift docker-env)`
+* Test the connection by running the following command:
+```code
+$ docker ps
+```
+
+   If successful, the shell will print a list of running containers.
+
+
+**Now we are ready to pull image**
+* login to `console.pathfinder.gov.bc.ca` 
+```command
+docker login docker-registry.pathfinder.gov.bc.ca -u $(oc whoami) -p $(oc whoami -t)
+docker pull docker-registry.pathfinder.gov.bc.ca/openshift/nodejs:10
+```
+some pre-work before push image to minishift
+
+```command
+docker tag docker-registry.pathfinder.gov.bc.ca/openshift/nodejs:10 nodejs:10
+docker build -t nodejs:10
```

Comment:
This infers a `Dockerfile` has been created but there are no instructions on creating a Dockerfile :)

Reply:
I add the instruction at line 83